docs/content/en/docs/implementing/evaluations.md

```diff
@@ -3,3 +3,85 @@ title: Evaluations
 description: Understand Keptn evaluations and how to use them
 weight: 150
 ---
+A `KeptnEvaluation` is a `KeptnMetric` that has a defined target value.
+A `KeptnEvaluation` resource is defined in a
+[KeptnEvaluationDefinition](../yaml-crd-ref/evaluationdefinition.md)
+yaml file.
+The example
+[app-pre-deploy-eval.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/version-3/app-pre-deploy-eval.yaml)
+file specifies the `app-pre-deploy-eval-2` evaluation as follows:
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnEvaluationDefinition
+metadata:
+  name: app-pre-deploy-eval-2
+  namespace: podtato-kubectl
+spec:
+  objectives:
+    - keptnMetricRef:
+        name: available-cpus
+        namespace: podtato-kubectl
+      evaluationTarget: ">4"`
+```
+
+The `evaluationTarget` is set to be `>4`,
+so this evaluation ensures that more than 4 CPUs are available
+before the workload or application is deployed.
+
+The example
+[metric.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/base/metric.yaml)
+file defines the
+[KeptnMetric](../yaml-crd-ref/metric.md) resource
+that is named  `available-cpus`:
+
+```yaml
```

Comment:
same as above

Reply:
Done, using the file that goes with this evaluationdefinition.